src/pages/en/guides/migrate-to-astro.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Migrate an existing site to Astro
+description: Some tips and tricks for converting your site to Astro.
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import CMSGuidesNav from '~/components/CMSGuidesNav.astro';
+i18nReady: true
+---
+
+SARAH ROUGH NOTES
+
+**Ready to convert your site to Astro?** Follow one of our guides to migrate from a different project/framework.
+
+## Migration Guides
+
+<CMSGuidesNav />
+
+Note that many of these pages are **stubs**: they're collections of resources waiting for your contribution!
+
+## Why migrate your site to Astro?
+
+Astro provides many benefits: performance, simplicity and several built-in and available integrations!
+
+Depending on your existing project, you may be able to bring some, or much of your existing files. However, Astro's project structure is likely to be different than the one you're using. So, there are some key concepts that will help you make the transition to Astro.
+
+
+## Which projects can I convert to Astro?
+
+Many existing sites can be built with Astro! Astro is ideally suited for your existing content-based sites like blogs, landing pages, marketing sites and portfolios. Astro integrates with several popular headless CMSs, and allows you to connect eCommerce shop carts.
+
+Astro allows you to choose between a statically-generated site and server-side rendering (SSR), making it a great replacement for SSGs or for sites that need to fetch some page data on the fly.
+
+Because Astro's focus is shipping as little JavaScript as necessary [via its island architecture](/en/concepts/islands/#what-is-an-astro-island), it is well-suited to sites where client-side interactivity is localized to particular components.
+
+## What changes will I encounter when moving to an island architecture?
+
+Migrating your site to Astro means thinking about your site in "islands" or isolated sections: which parts are static? which are dynamic? which are interactive? which ones need to communicate with others?
+
+Thinking about your site as "islands" may be a shift of thinking, which brings with it some additional considerations:
```

Comment:
The following suggestion was originally written by @sarah11918:

```suggestion
As you shift your thinking away from a default shared app-level context and towards "islands by default" in Astro, you'll design differently:
```

Totally not sold on how I worded the "away from", especially since they could be moving "Away from" any number of things!" and honestly could even drop it. BUT, if there's a clear, concise drop in for "away from...." that is representative of all non-island architecture thinking, I think I'd like to include it in a sentence like this.

Using a simpler verb tense here, I'm also now less sold on the line above being in the "Migrating..." verb tense. ("When you migrate your site to Astro, you'll now think about...") So I'm totally open to changing that, too!

Reply:
I really like the "You'll design differently" line, but think the "default shared app-level context" is a bit terminology-heavy.

Maybe we could try:

```suggestion
As you shift your thinking away from "all JavaScript all the time" and towards "islands by default" in Astro, you'll design differently:
```

It's a little oversimplified, but it may speak better to engineers like me who struggle with terminology overload?

What do you think @sarah11918

Reply:
Approved!